tool/travis.sh

```diff
@@ -4,9 +4,6 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-# Fast fail the script on failures.
-set -e
```

Comment:
Need to keep this. Drop "set +e" and "return 0" at line 44, 46.

Reply:
Done.